### DISCUSSION TITLE
NEW - Supply Source API (with docs from Amazon Architects)

### DISCUSSION THREAD

**Discussion by owenashurst** [abuzuhri/Amazon-SP-API-CSharp#662] [General]
Hi all,

I have new API documentation for a new API Amazon have created but isn't documented on their website yet for their new in store collect shipping method. This allows users to purchase an item and then go and pick up the item.

I personally don't have time to implement this, however I am looking to use this library for this purpose very soon.

I've attached the API Docs I received from the Amazon Architects I spoke to.

Would anyone be interested in picking this up please?

[BOPIS_Supply Source API - additional details.pdf](https://github.com/abuzuhri/Amazon-SP-API-CSharp/files/12709590/BOPIS_Supply.Source.API.-.additional.details.pdf)
[SampleRequest-Response.docx](https://github.com/abuzuhri/Amazon-SP-API-CSharp/files/12709591/SampleRequest-Response.docx)
[Supply Source API Guide v3.docx](https://github.com/abuzuhri/Amazon-SP-API-CSharp/files/12709592/Supply.Source.API.Guide.v3.docx)

**Comment by clementcampagna** [comment]
Hi there,

I'm also looking into Amazon's new BOPIS solution.

Here are three documents explaining how it works :

[BOPIS Order Management.pdf](https://github.com/abuzuhri/Amazon-SP-API-CSharp/files/12814182/BOPIS.Order.Management.pdf)
[BOPIS Store Management.pdf](https://github.com/abuzuhri/Amazon-SP-API-CSharp/files/12814183/BOPIS.Store.Management.pdf)
[BOPIS Store Inventory Management.pdf](https://github.com/abuzuhri/Amazon-SP-API-CSharp/files/12814184/BOPIS.Store.Inventory.Management.pdf)

Unfortunately, and like @owenashurst , I do not have time to implement this into **Amazon-SP-API-CSharp** right now.

Would @abuzuhri or someone else feel like doing it?

If so, please let us know, I'd be happy to help whenever I can and do some beta testing.

Thanks.
